ckanext/security/plugin/__init__.py

```diff
@@ -106,4 +106,5 @@ def get_helpers(self):
         return {
             'check_ckan_version': tk.check_ckan_version,
             'security_enable_totp': security_enable_totp,
+            'security_get_user': security_get_user,
```

Comment:
Can this have a more descriptive name?

Reply:
Perhaps something more descriptive of its purpose, eg `get_user_by_name_or_email`

Reply:
I have changed the function name from security_get_user to security_get_user_by_name_or_email.